src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanOrchestrator.hpp

```diff
@@ -270,8 +270,11 @@ class TScanOrchestrator final : public TOSPrimitives
              const rocksdb::PinnableSlice& rawData,
              bool isDelayed) const
     {
-        // The scan only reads the content
-        std::shared_lock lock(m_mutex);
+        // The scan only reads the content.
```

Comment:
why do we do this? create a new mutex? Isn't it enough to make it scoped lock like the one we had?

Reply:
We wanted to be clear about the intention of the code.
We talked about that possibility in the daily, but we decided to use a more expressive approach.

I'll use the same mutex.